homeassistant/components/supla/__init__.py

```diff
@@ -0,0 +1,165 @@
+"""
+Support for the Supla devices.
+
+For more details about this platform, please refer to the documentation.
+https://home-assistant.io/components/supla/
+"""
+
+import logging
+from typing import Optional
+import voluptuous as vol
```

Comment:
Please keep the imports [grouped](https://developers.home-assistant.io/docs/en/development_guidelines.html#ordering-of-imports).

Reply:
Imports sorted. Thanks for the suggestion!